tests/unit/Term/AliasGroupListTest.php

```diff
@@ -170,20 +171,13 @@ public function testGivenInvalidLanguageCode_removeByLanguageThrowsException( $l
 	}
 
 	public function testGivenEmptyGroups_constructorRemovesThem() {
-		$enGroup = new AliasGroup( 'en', array( 'foo' ) );
-
 		$list = new AliasGroupList( array(
 			new AliasGroup( 'de' ),
-			$enGroup,
-			new AliasGroup( 'en' ),
-			new AliasGroup( 'nl' ),
-		) );
-
-		$expectedList = new AliasGroupList( array(
+			new AliasGroup( 'en', array( 'foo' ) ),
 			new AliasGroup( 'en' ),
 		) );
 
-		$this->assertEquals( $expectedList, $list );
+		$this->assertEquals( new AliasGroupList(), $list );
```

Comment:
Here is a good one: What's the most minimal implementation to make this test succeed?
```php
public function testGivenSetLanguage_removeByLanguageRemovesIt() {
	$list = new AliasGroupList( array( new AliasGroup( 'en', array( 'foo', 'bar' ) ) ) );

	$list->removeByLanguage( 'en' );

	$this->assertEquals( new AliasGroupList(), $list );
}
```
I will show you:
```php
class AliasGroupList {
	public function removeByLanguage( $languageCode ) {
	}
}
```